clickplc/__init__.py

```diff
@@ -17,15 +17,20 @@ def command_line():
     parser = argparse.ArgumentParser(description="Control a ClickPLC from "
                                      "the command line.")
     parser.add_argument('address', help="The IP address of the ClickPLC.")
+    parser.add_argument('tags_file', default=None,
+                        help="Optional: Path to a tags file for this PLC" )
```

Comment:
Add a . to match others, or delete other . to match this

Reply:
👍